.gitignore

```diff
@@ -60,6 +60,7 @@ default_storage_unit_core
 default_storage_unit_storage
 test.ipynb
 test2.ipynb
+test3.ipynb
```

Comment:
Are you spamming the gitignore with your test notebooks? :D

Reply:
Yes, because I have a few POCs that I don't want to delete. I'll remove this one though :)